src/librbd/internal.cc

```diff
@@ -72,6 +73,19 @@ namespace librbd {
 
 namespace {
 
+struct C_CreateV2Image : public Context {
```

Comment:
Not needed -- pass in the pointer to a stack-allocated ```C_SaferCond``` for the ```Context``` and use the ```wait``` function to retrieve the result.